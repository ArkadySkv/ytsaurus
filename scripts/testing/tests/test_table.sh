#!/bin/bash 
#% NUM_MASTERS = 3
#% NUM_HOLDERS = 5

yt create /table table

echo 'read empty table'
yt read /table
yt get /table
yt get /table@row_count

echo 'write one value'
yt write /table [{b="hello"}]
yt get /table@row_count
yt read /table

echo 'write many values'
yt write /table '[{b="2";a="1"};{x="10";y="20";a="30"}]'
yt get /table@row_count
yt read /table

#TODO(panin): add more checks of read methods

