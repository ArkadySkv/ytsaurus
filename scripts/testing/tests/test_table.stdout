{"object_id"="0-1-191-269b57e6"}
<>
0
write one value
1
{"b"="hello"}
write many values
3
{"b"="hello"}
{"a"="1";"b"="2"}
{"a"="30";"x"="10";"y"="20"}
write from stream
4
{"b"="hello"}
{"a"="1";"b"="2"}
{"a"="30";"x"="10";"y"="20"}
{"row"="some"}
